### PR TITLE
[Bots][Conversationtype][1662450]

### PR DESCRIPTION
**Pull request by v-rodriguezd**
In first table changed teams to channel as per issue 2932 and grammar and language changes. Please review.

### CODE REVIEW

**Comment by KirtiPereira** [comment]
@v-rodriguezd - please resolve the warning.

File: msteams-platform/bots/how-to/conversations/conversation-basics.md [hunk 1]
```diff
@@ -5,38 +5,45 @@ ms.topic: overview
...
 
-A conversation is a series of messages sent between your bot and one or more users. There are three types of conversations, also called scopes in Teams:
+A conversation is a series of messages sent between your Microsoft Teams bot and one or more users. There are three types of conversations, also called scopes in Teams:
 
 | Conversation type | Description |
 | ------- | ----------- |
-|  `teams` | Also called channel conversations, visible to all members of the channel. |
-| `personal` | Conversations between bots and a single user. |
-| `groupChat` | Chat between a bot and two or more users. Also enables your bot in meeting chats. |
+| `channel` | This conversation type is also called channel conversations, visible to all members of the channel. |
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/conversations/conversation-basics.md:17]: This is repetitive.

File: msteams-platform/bots/how-to/conversations/conversation-basics.md [hunk 2]
```diff
@@ -207,29 +216,33 @@ async def on_members_added_activity(
...
+
+Messages sent between users and bots include internal channel data within the message. This data allows the bot to communicate properly on that channel. The Bot Builder SDK allows you to modify the message structure.
 
 ## Teams channel data
 
-The `channelData` object contains Teams-specific information and is a definitive source for team and channel IDs. You may need to cache and use these IDs as keys for local storage. The `TeamsActivityHandler` in the SDK, typically pulls out important information from the `channelData` object to make it easily accessible. However, you can always access the original data from the `turnContext` object.
+The `channelData` object contains Teams-specific information and is a definitive source for team and channel IDs. You must cache and use these IDs as keys for local storage. The `TeamsActivityHandler` in the SDK typically pulls out important information from the `channelData` object to make it easily accessible. However, you can always access the original data from the `turnContext` object.
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/conversations/conversation-basics.md:225]: This is not a required step. Could you please change this?

File: msteams-platform/bots/how-to/conversations/conversation-basics.md [hunk 3]
```diff
@@ -373,13 +398,45 @@ Use the following code to send a simple adaptive card:
 
 To know more about cards and cards in bots, see [cards documentation](~/task-modules-and-cards/what-are-cards.md).
 
+The next section provides status code responses for errors generated from Bot APIs.
+
+## Status code responses
+
+The following image shows a sample 403 status code response:
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/conversations/conversation-basics.md:405]: We can remove this image.

File: msteams-platform/bots/how-to/conversations/conversation-basics.md [hunk 4]
```diff
@@ -373,13 +398,45 @@ Use the following code to send a simple adaptive card:
...
+| 403 | **Code**: `BotNotInConversationRoster` <br/> **Message**: "The bot is not part of the conversation roster." | The bot is not part of the conversation. |
+| 403 | **Code**: `BotDisabledByAdmin` <br/> **Message**: "The tenant admin disabled this bot." | Tenant blocked the bot. |
+| 401 | **Code**: `BotNotRegistered` <br/> **Message**: “No registration found for this bot.” | The registration for this bot was not found. |
+| 412 | **Code**: `PreconditionFailed` <br/> **Message**: “Precondition failed, please try again.” | A precondition failed on one of our dependencies due to multiple concurrent operations on the same conversation. |
+| 404 | **Code**: `ConversationNotFound` <br/> **Message**: “Conversation not found.” | The conversation was not found. |
+| 413 | **Code**: `MessageSizeTooBig` <br/> **Message**: “Message size too large.” | The size on the incoming request was too large. |
+| 429 | **Code**: `Throttled` <br/> **Message**: “Too many requests.” Also returns when to retry after. | Too many requests were sent by the bot. |
```
Comment (Wajeed-msft) [inline msteams-platform/bots/how-to/conversations/conversation-basics.md:420]: Let's add link to https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/rate-limit

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: msteams-platform/bots/how-to/conversations/conversation-basics.md (hunk 1 above)
Comment (v-rodriguezd) [reply msteams-platform/bots/how-to/conversations/conversation-basics.md:17]: I did not understand what is repetitive. Please elaborate.

**Review by Wajeed-msft — APPROVED** [review]

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
approved PR

**Pull request merged**